vignettes/02-get-weather-data-R.Rmd

```diff
@@ -117,34 +47,26 @@ daily_values = weather_data %>%
                          (max_temp + min_temp) / 2 - 10, 0))
 ```
 
-### Plot data using R
+## Plot data using R
 
-The five minute summary weather variables in the `weather_data` dataframe can be 
-plotted across time, as shown below for temperature and precipitation. 
+The five minute summary weather variables in the `weather_data` dataframe can be plotted across time, as shown below for temperature.  
 
 ```{r temp-plot}
-theme_set(ggthemes::theme_few())
 ggplot(data = weather_data) +
   geom_point(aes(x = time, y = air_temperature), size = 0.1) +
   labs(x = "Date", y = "Temperature (K)")
 ```
 
-```{r precip-plot}
-ggplot(data = weather_data) +
-  geom_point(aes(x = time, y = precipitation_rate), size = 0.1) +
-  labs(x = "Date", y = "Precipitation (kg/m2s)")
-```
-
-### Get all available weather variables
+We can also plot the time series for all eight of the weather variables in a single figure. 
+We first have to rearrange the data to making plotting possible using R package `ggplot`. 
 
-The weather variables that are available from these datapoints data are 
-extracted below from the column names of the dataframe that we read in earlier. 
-Any of these variables that are of interest can be analyzed and plotted. 
+```{r}
+weather_data_long = weather_data %>% select(-source, -source_file) %>% gather(weather_variable, observation, -time)
```

Comment:
I'd break up this statement between the pipes (consistent with your met-GDD chunk)